src/approvals-service.ts

```diff
@@ -308,7 +308,7 @@ export async function getFormApprovalFlowInstanceSubmission(
 
 type FormApprovalFlowResponse = {
   forms: FormTypes.Form[]
-  formApprovalFlows: ApprovalTypes.FormApprovalFlow[]
+  formApprovalFlows: ApprovalTypes.FormApprovalFlowInstance[]
```

Comment:
Im actually going to remove this property from the response. So it will just be:

```typescript
type FormApprovalFlowResponse = {
  forms: FormTypes.Form[]
}
```